packages/data-stores/src/launch/selectors.ts

```diff
@@ -22,7 +22,7 @@ export const hasPaidDomain = ( state: State ): boolean => {
 	if ( ! state.domain ) {
 		return false;
 	}
-	return ! state.domain.is_free;
+	return ! state.domain.is_free; // @TODO: check if we are ever storing a paid domain
```

Comment:
Is this `@TODO` still applicable? If so, should it be solved in this PR or should we open a follow-up issue?

Reply:
It's still applicable because this check for free domain is not needed with current Launch logic. If the user selects the sub-domain (free domain), we don't store it but we unset the domain selection. See https://github.com/Automattic/wp-calypso/blob/trunk/packages/launch/src/hooks/use-domain-selection.ts#L117

Reply:
Tracked separately in https://github.com/Automattic/wp-calypso/issues/51860